components/post_view/channel_intro_message/channel_intro_message.tsx

```diff
@@ -351,17 +360,22 @@ export function createDefaultIntroMessage(channel: Channel, centeredIntro: strin
                 }
             </p>
             {teamInviteLink}
-            {setHeaderButton}
+            {teamIsGroupConstrained && setHeaderButton}
```

Comment:
Is this condition `teamIsGroupConstrained` here needed? Noticed it wasn't set before, I may not have the complete context

Reply:
yes, the condition is needed, because the setHeader button was pass as a prop to the AddMembersButton component, so this way we avoid having two buttons or duplicating validation logic inside the AddMembersButton component.